playwright/src/test/java/com/microsoft/playwright/TestBrowserContextAddCookies.java

```diff
@@ -40,34 +38,6 @@ void shouldWork() {
     assertEquals("password=123456", page.evaluate("document.cookie"));
   }
 
-  @Test
-  void shouldRoundtripCookie() {
```

Comment:
This test is actually present upstream and seems that https://github.com/microsoft/playwright-java/pull/1045/ actually changed the tests logic a bit which makes it now indistinguishable from shouldGetANonSessionCookie. Let's align this test with upstream instead of deleting it:

```java
    List<Cookie> cookies = context.cookies();
    assertEquals(1, cookies.size());
    context.clearCookies();
    assertEquals(emptyList(), context.cookies());
    context.addCookies(cookies);
    assertJsonEquals(new Gson().toJson(cookies), context.cookies());```